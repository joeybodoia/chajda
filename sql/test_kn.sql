CREATE EXTENSION IF NOT EXISTS pspacy;
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
SELECT spacy_tsvector('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."');
