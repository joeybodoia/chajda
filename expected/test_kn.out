CREATE EXTENSION IF NOT EXISTS pspacy;
NOTICE:  extension "pspacy" already exists, skipping
SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                       spacy_lemmatize                                       
---------------------------------------------------------------------------------------------
   ಪರೀಕ್ಷಾ:2 ಪ್ರಕರಣಗಳನ್ನು:3 ರಚಿಸಲು:4 google:6 ಅನುವಾದಕ್ಕೆ:7 ಹಾಕುತ್ತಿರುವ:8 ಉದಾಹರಣೆ:10 ಪರೀಕ್ಷಾ:11 ವಾಕ್ಯ:12
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                            spacy_lemmatize                             
------------------------------------------------------------------------
   ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                 spacy_lemmatize                                                 
-----------------------------------------------------------------------------------------------------------------
   ಪರೀಕ್ಷಾ:2 ಪ್ರಕರಣಗಳನ್ನು:3 ರಚಿಸಲು:4 ನಾನು:5 google:6 ಅನುವಾದಕ್ಕೆ:7 ಹಾಕುತ್ತಿರುವ:8 ನನ್ನ:9 ಉದಾಹರಣೆ:10 ಪರೀಕ್ಷಾ:11 ವಾಕ್ಯ:12 ಇದು:13
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                   spacy_lemmatize                                   
-------------------------------------------------------------------------------------
   ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                              spacy_lemmatize                                              
-----------------------------------------------------------------------------------------------------------
   ":2 ಪರೀಕ್ಷಾ:3 ಪ್ರಕರಣಗಳನ್ನು:4 ರಚಿಸಲು:5 google:7 ಅನುವಾದಕ್ಕೆ:8 ಹಾಕುತ್ತಿರುವ:9 ಉದಾಹರಣೆ:11 ಪರೀಕ್ಷಾ:12 ವಾಕ್ಯ:13 .:15 ":16
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                               spacy_lemmatize                                
------------------------------------------------------------------------------
   " ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ . "
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                        spacy_lemmatize                                                         
--------------------------------------------------------------------------------------------------------------------------------
   ":2 ಪರೀಕ್ಷಾ:3 ಪ್ರಕರಣಗಳನ್ನು:4 ರಚಿಸಲು:5 ನಾನು:6 google:7 ಅನುವಾದಕ್ಕೆ:8 ಹಾಕುತ್ತಿರುವ:9 ನನ್ನ:10 ಉದಾಹರಣೆ:11 ಪರೀಕ್ಷಾ:12 ವಾಕ್ಯ:13 ಇದು:14 .:15 ":16
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>True , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                      spacy_lemmatize                                      
-------------------------------------------------------------------------------------------
   " ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು . "
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>True);
                                       spacy_lemmatize                                       
---------------------------------------------------------------------------------------------
   ಪರೀಕ್ಷಾ:2 ಪ್ರಕರಣಗಳನ್ನು:3 ರಚಿಸಲು:4 Google:6 ಅನುವಾದಕ್ಕೆ:7 ಹಾಕುತ್ತಿರುವ:8 ಉದಾಹರಣೆ:10 ಪರೀಕ್ಷಾ:11 ವಾಕ್ಯ:12
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>True , add_positions=>False);
                            spacy_lemmatize                             
------------------------------------------------------------------------
   ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>True);
                                                 spacy_lemmatize                                                 
-----------------------------------------------------------------------------------------------------------------
   ಪರೀಕ್ಷಾ:2 ಪ್ರಕರಣಗಳನ್ನು:3 ರಚಿಸಲು:4 ನಾನು:5 Google:6 ಅನುವಾದಕ್ಕೆ:7 ಹಾಕುತ್ತಿರುವ:8 ನನ್ನ:9 ಉದಾಹರಣೆ:10 ಪರೀಕ್ಷಾ:11 ವಾಕ್ಯ:12 ಇದು:13
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>True , remove_stop_words=>False , add_positions=>False);
                                   spacy_lemmatize                                   
-------------------------------------------------------------------------------------
   ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>True);
                                              spacy_lemmatize                                              
-----------------------------------------------------------------------------------------------------------
   ":2 ಪರೀಕ್ಷಾ:3 ಪ್ರಕರಣಗಳನ್ನು:4 ರಚಿಸಲು:5 Google:7 ಅನುವಾದಕ್ಕೆ:8 ಹಾಕುತ್ತಿರುವ:9 ಉದಾಹರಣೆ:11 ಪರೀಕ್ಷಾ:12 ವಾಕ್ಯ:13 .:15 ":16
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>True , add_positions=>False);
                               spacy_lemmatize                                
------------------------------------------------------------------------------
   " ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ . "
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>True);
                                                        spacy_lemmatize                                                         
--------------------------------------------------------------------------------------------------------------------------------
   ":2 ಪರೀಕ್ಷಾ:3 ಪ್ರಕರಣಗಳನ್ನು:4 ರಚಿಸಲು:5 ನಾನು:6 Google:7 ಅನುವಾದಕ್ಕೆ:8 ಹಾಕುತ್ತಿರುವ:9 ನನ್ನ:10 ಉದಾಹರಣೆ:11 ಪರೀಕ್ಷಾ:12 ವಾಕ್ಯ:13 ಇದು:14 .:15 ":16
(1 row)

SELECT spacy_lemmatize('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."' , lower_case=>False , remove_special_chars=>False , remove_stop_words=>False , add_positions=>False);
                                      spacy_lemmatize                                      
-------------------------------------------------------------------------------------------
   " ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು . "
(1 row)

SELECT spacy_tsvector('kn',' "ಪರೀಕ್ಷಾ ಪ್ರಕರಣಗಳನ್ನು ರಚಿಸಲು ನಾನು Google ಅನುವಾದಕ್ಕೆ ಹಾಕುತ್ತಿರುವ ನನ್ನ ಉದಾಹರಣೆ ಪರೀಕ್ಷಾ ವಾಕ್ಯ ಇದು."');
                                           spacy_tsvector                                           
----------------------------------------------------------------------------------------------------
 'google':6 'ಅನುವಾದಕ್ಕೆ':7 'ಉದಾಹರಣೆ':10 'ಪರೀಕ್ಷಾ':2,11 'ಪ್ರಕರಣಗಳನ್ನು':3 'ರಚಿಸಲು':4 'ವಾಕ್ಯ':12 'ಹಾಕುತ್ತಿರುವ':8
(1 row)

